Relative imports not working on Windows

Problem is related to #145 issue. That issues added a function IsPathRelative(s string) in interp/src.go file. Problem is that os specific file separator is used insted of "/", so it doesn't work correctly on windows.
So I would suggest changing of line:
    p := "." + string(filepath.Separator)
to:
    p := "./"

Best wishes, Franis.

The following program `sample.go` triggers a panic:

```go
package main

import (
	"./footrt"
)

func Bar(args map[string]interface{}) error {
	println(args)
	return nil
}
```

Expected result:
```console
$ go run ./sample.go
not applicable

Got:
```console
$ yaegi ./sample.go
run: .\sample.go:4:2: import "./footrt" error: package location C:\Users\franis\go\src\github.com\traefik\yaegi not in GOPATH
```

--- a/interp/src.go
+++ b/interp/src.go
@@ -303,7 +303,7 @@ func effectivePkg(root, path string) string {
 }
 
 // isPathRelative returns true if path starts with "./" or "../".
+// It is intended for use on import paths, where "/" is always the directory separator.
 func isPathRelative(s string) bool {
-	p := "." + string(filepath.Separator)
-	return strings.HasPrefix(s, p) || strings.HasPrefix(s, "."+p)
+	return strings.HasPrefix(s, "./") || strings.HasPrefix(s, "../")
 }

